fink_science/random_forest_snia/classifier_sigmoid.py

```diff
@@ -291,6 +291,30 @@ def get_train_test(percent_train):
 
     return sample
 
+def average_intraday_data(df_intra):
+
+    """Average over intraday data points
+
+     Parameters
+     ----------
+     df_intra: pd.DataFrame
+        containing the history of the flux
+        with intraday data
+
+     Returns
+     -------
+     df_average: pd.DataFrame
+        containing only daily data
+    """
+
+    df_average = df_intra.copy()
+    df_average['MJD'].values = df_average['MJD'].apply(
+        lambda x: np.around(x, decimals=0))

```

Comment:
Marco - this is still failing because left hand side and right hand side have not the same type (you left `values` on the left hand side). I suggested:

```python
df_average['MJD'] = df_average['MJD'].apply(lambda x: np.around(x, decimals=0))
```
and not
```python
df_average['MJD'].values = df_average['MJD'].apply(lambda x: np.around(x, decimals=0))
```
as it is currently.

Reply:
Ah, shit, sorry -- my bad. This is now corrected. But I still can't manage to run tests in my local repository.

Reply:
Thanks!